src/System.Threading/src/System.Threading.csproj

```diff
@@ -29,7 +29,7 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
-      <OSGroup>Windows_NT</OSGroup>
+      <OSGroup>$(TestOsTarget)</OSGroup>
```

Comment:
TestOSTarget shouldn't be used in the product builds. Is there are reason you needed to update this?